README.md

```diff
@@ -96,7 +96,17 @@ FAR CRs are deleted by NHC after it sees the Node is healthy again.
 
 #### Example CR
 
-The FAR, `FenceAgentsRemediation`, CR is created by the admin and is used to trigger the fence-agent on a specific node. The CR includes an *agent* field for the fence-agent name, *sharedparameters* field with all the shared, not specific to a node, parameters, and *nodeparameters* field to specify the parameters for the fenced node.
+The FAR CR (CustomResource), created by the admin, is used to trigger the fence-agent on a specific node.
+
+The CR includes the following parameters:
+
+* `agent` - fence-agent name
+* `sharedparameters` - common cluster nodes information for executing the fence agent
```

Comment:
I'm not 100% sure if I have the meaning of these CR fields right in my head... but from here I don't really get it 🤔 

IIUC:

- `sharedparameters` - cluster wide parameters for executing the fence agent
- `nodeparameters` - node specific parameters for executing the fence agent

WDYT? 
@clobrano @razo7

Reply:
It is kind of similar.
I think `common cluster nodes information` is just saying in different words that it is `cluster wide parameters` without repeating the word `parameters` for explaining `sharedparameters`.
I don't have a strong preference, even though your suggestion might be just more accurate.

Reply:
I like @slintes version too, thanks!